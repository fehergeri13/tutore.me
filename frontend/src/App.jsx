import React from 'react';


import "./app.less"

export default class App extends React.Component {
    render() {
        return <div className="app">
            <div className="header">
                <h1>tutore.me</h1>

                <div className="nav">
                    <a href="#">Hirdetések</a>
                    <a href="#">Új hirdetés</a>
                </div>

                <div className="user">
                    <a href="#">Üzenetek</a>
                    <a href="#">Profilom</a>
                    <a href="#">Kilépés</a>
                </div>
            </div>

            <div className="search">
                <input type="text" id="search-text" placeholder="keresés"/>

                <fieldset>
                    <label>Keres<input type="radio" name="contact" value="keres"/></label>
                    <label>Kínál<input type="radio" name="contact" value="keres"/></label>
                </fieldset>

                <fieldset>
                    <legend>
                        Tantárgy
                    </legend>
                    <label>informatika<input type="checkbox" name="contact" value="keres"/></label>
                    <label>Matematika<input type="checkbox" name="contact" value="keres"/></label>
                    <label>Fizika<input type="checkbox" name="contact" value="keres"/></label>
                    <label>Kémia<input type="checkbox" name="contact" value="keres"/></label>
                </fieldset>
            </div>

            <div className="content">
                <ul>
                    <li>
                        <h2><a href="#">Matematika korrepetálás</a></h2>
                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Laudantium nisi perspiciatis voluptas. A aut consectetur deleniti, ea impedit laboriosam minima molestias non numquam, placeat tempore temporibus ullam. Itaque, provident quae?</p>
                    </li>
                    <li>
                        <h2><a href="#">Matematika korrepetálás</a></h2>
                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Laudantium nisi perspiciatis voluptas. A aut consectetur deleniti, ea impedit laboriosam minima molestias non numquam, placeat tempore temporibus ullam. Itaque, provident quae?</p>
                    </li>
                    <li>
                        <h2><a href="#">Matematika korrepetálás</a></h2>
                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Laudantium nisi perspiciatis voluptas. A aut consectetur deleniti, ea impedit laboriosam minima molestias non numquam, placeat tempore temporibus ullam. Itaque, provident quae?</p>
                    </li>
                    <li>
                        <h2><a href="#">Matematika korrepetálás</a></h2>
                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Laudantium nisi perspiciatis voluptas. A aut consectetur deleniti, ea impedit laboriosam minima molestias non numquam, placeat tempore temporibus ullam. Itaque, provident quae?</p>
                    </li>
                    <li>
                        <h2><a href="#">Matematika korrepetálás</a></h2>
                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Laudantium nisi perspiciatis voluptas. A aut consectetur deleniti, ea impedit laboriosam minima molestias non numquam, placeat tempore temporibus ullam. Itaque, provident quae?</p>
                    </li>
                </ul>
            </div>
        </div>;
    }
}
