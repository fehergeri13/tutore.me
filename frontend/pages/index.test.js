/* eslint-env jest */

import { mount } from 'enzyme'
import React from 'react'
import renderer from 'react-test-renderer'

import App from './index.js'

describe('With Enzyme', () => {
  it('App shows "Index Page"', () => {
    const app = mount(<App />)

    expect(app.find('h1').text()).toEqual('Index Page')
  })
})

describe('With Snapshot Testing', () => {
  it('App shows "Index Page!"', () => {
    const component = renderer.create(<App />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})