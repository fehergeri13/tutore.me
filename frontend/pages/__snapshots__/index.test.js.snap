// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing App shows "Index Page!" 1`] = `
<div>
  <h1>
    Index Page
  </h1>
  <div
    className="jsx-3493801758 "
  >
    23:36:07
  </div>
  <nav>
    <a
      href="/other"
      onClick={[Function]}
    >
      Navigate
    </a>
  </nav>
</div>
`;
